### PR TITLE
BT-26158: Fixed session mail placeholders

### PR DESCRIPTION
**Pull request by M-Barz**
Link to Mantis-Report:
https://mantis.ilias.de/view.php?id=26158

### CODE REVIEW

File: Modules/Session/classes/class.ilSessionMailTemplateParticipantContext.php [hunk 1]
```diff
@@ -98,20 +98,25 @@ public function resolveSpecificPlaceholder(
...
+        $obj_id = $ilObjDataCache->lookupObjId($context_parameters['ref_id']);
+        $sess_data = ilObjSession::lookupSession($obj_id);
+        $sess_app = ilSessionAppointment::_lookupAppointment($obj_id);
+
+        switch ($placeholder_id) {
+            case 'sess_title':
+                return $ilObjDataCache->lookupTitle($obj_id);
+            case 'sess_appointment':
+                return ilDatePresentation::formatPeriod(
+                    new ilDateTime($sess_app['start'], IL_CAL_UNIX),
```
Comment (smeyer-ilias) [inline Modules/Session/classes/class.ilSessionMailTemplateParticipantContext.php]: This always presents dates with time information.
Please use ilDate for fullday events and ilDateTime for non fullday  events.

**Review by smeyer-ilias — CHANGES_REQUESTED** [review]

**Comment by M-Barz** [comment]
Good call! It's now working for both cases :)

File: Modules/Session/classes/class.ilSessionMailTemplateParticipantContext.php (hunk 1 above)
Comment (M-Barz) [reply Modules/Session/classes/class.ilSessionMailTemplateParticipantContext.php]: Good call! It's now working for both cases :)

**Review by smeyer-ilias — APPROVED** [review]
Thanks, please also merge to 5.4

**Pull request merged**